component.json

```diff
@@ -59,6 +101,47 @@
         "out": {}
       }
     },
+    "read_action_v2": {
+      "main": "./lib/triggers/read_v2.js",
+      "title": "Read CSV attachment v2",
+      "fields": {
+        "url": {
+          "label": "CSV URL",
+          "required": false,
+          "placeholder": "http://my-url.com/foo.csv",
```

Comment:
Perhaps use a steward URL as the placeholder?

Reply:
There is no placeholder after field moved to metadata.